src/volatile-root/volatile-root.c

```diff
@@ -18,6 +18,9 @@
 #include "string-util.h"
 #include "volatile-util.h"
 
+const char *overlay_root_name = "overlay-root";
+const char *overlay_usr_name = "overlay-usr";
```

Comment:
Please don't use (mutable) global variables here. If you want, use `#define`. But just using the string directly in the function where it is used once is fine.